sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3026,7 +3026,7 @@ _ZN4sycl3_V13ext6oneapi12experimental20pitched_alloc_deviceEPmRKNS3_16image_desc
 _ZN4sycl3_V13ext6oneapi12experimental20pitched_alloc_deviceEPmRKNS3_16image_descriptorERKNS0_6deviceERKNS0_7contextE
 _ZN4sycl3_V13ext6oneapi12experimental20pitched_alloc_deviceEPmmmjRKNS0_5queueE
 _ZN4sycl3_V13ext6oneapi12experimental20pitched_alloc_deviceEPmmmjRKNS0_6deviceERKNS0_7contextE
-_ZN4sycl3_V13ext6oneapi12experimental21dynamic_command_group14set_active_cgfEm
+_ZN4sycl3_V13ext6oneapi12experimental21dynamic_command_group16set_active_indexEm
```

Comment:
dynamic_command_group class was introduced just recently https://github.com/intel/llvm/commit/5c73a20d4df4d9fd6e0e1f1594207c74b8f8d855 and so hasn't been included to any release yet.
Considering that  and that this is experimental extension I believe it is acceptable to break abi in this case.

Tagging @intel/llvm-gatekeepers in case if there is another opinion.